deepmd/descriptor/se_a.py

```diff
@@ -186,6 +190,15 @@ def __init__(
             self.exclude_types.add((tt[1], tt[0]))
         self.set_davg_zero = set_davg_zero
         self.type_one_side = type_one_side
+        self.spin = spin
+
+        # extend sel_a for spin system
+        if self.spin is not None:
+            self.ntypes_spin = self.spin.get_ntypes_spin()
+            self.sel_a_spin = self.sel_a[: self.ntypes_spin]
+            self.sel_a.extend(self.sel_a_spin)
```

Comment:
consider applying them to `se_r` and `hybrid`

Reply:
We have applied  them to `se_r` and `hybrid`.